src/xGadgetron/cGadgetron/gadgetron_data_containers.h

```diff
@@ -418,7 +420,7 @@ namespace sirf {
 
 	Images are stored in an std::vector<shared_ptr<ImageWrap> > object.
 	*/
-	class ImagesVector : public MRImageData {
+	class ImagesVector : public GadgetronImageData {
```

Comment:
if derived from `GadgetronImageData`, the class `ImagesVector` would have to change name.  Possibly could just be templated